src/commands/synthetics/run-test.ts

```diff
@@ -213,6 +221,27 @@ export class RunTestCommand extends Command {
     let hasSucceeded = true // Determine if all the tests have succeeded
     for (const test of tests) {
       const testResults = results[test.public_id]
+      if (!this.config.failOnTimeout) {
+        if (!summary.timedOut) {
+          summary.timedOut = 0
```

Comment:
No need to make `timedOut` and `criticalErrors` optional, we should instantiate both at 0 [here](https://github.com/DataDog/datadog-ci/blob/c05c7499663c1c6ac8cd08982dfe0f5ed78b39ef/src/commands/synthetics/utils.ts#L429) and if they are not incremented they wont be displayed anyway.